universe/ShipDesign.cpp

```diff
@@ -84,6 +84,52 @@ namespace {
         // have an ID yet) is the same as one that has already been added
         // (which does have an ID)
     }
+
+    constexpr unsigned int CHECKSUM_MODULUS = static_cast<unsigned int>(10'000'000);
```

Comment:
Apparently the Travis CI compiler doesn't support the `'` separator in numeric constants from C++14.